docker/dlang

```diff
@@ -11,6 +11,7 @@ set -xeu
 dist="$(lsb_release -cs)"
 
 # Add extra dlang APT repositories from Sociomantic in Bintray
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 379CE192D401AB61
```

Comment:
I think this should still be in a base image (because this  key applies to all bintray repos)

Reply:
Meh, but it's noise at this stage, I prefer to leave adding repos as an atomic operation. Even more, I think the download of the key should be done each time a repo is added, so we don't depend on how base images were built.

Reply:
Not sure if you remember, but adding extra repos was the very point of using our `base` image for deployment images as opposed to plain `ubuntu`. This was the only real difference.

Reply:
Yeah, but that will go to the new "base" image, that will only include bintray repos (not just the key) for all the base libraries needed run actually run stuff (as opposed to build it and test it).